cloud/aws/services/interfaces.go

```diff
@@ -70,4 +70,6 @@ type EC2MachineInterface interface {
 // ELBInterface encapsulates the methods exposed by the elb service.
 type ELBInterface interface {
 	ReconcileLoadbalancers(clusterName string, network *providerv1.Network) error
+	DeleteLoadbalancers(clusterName string, network *providerv1.Network) error
+	RegisterInstanceWithAPIServerELB(clusterName string, instanceID string) error
```

Comment:
Should we simplify this method by passing in a load balancer name/id instead?

Reply:
That'd be better I think, the method names in the load balancer service did surprising things (like only ever creating an API Server load balancer).

Needs a bit of a rewrite I think.